Design rework

nizhgo opened the pull request:
Redesign has been made. The site after this commit will look like this: https://ibb.co/pWFjwMS

theyllneverfindyourgrave commented:
I really like it! I vote for the merging, but I'll wait for the @the2pizza desicion.

theyllneverfindyourgrave commented:
But actually I would use social media icons instead of those buttons or atleast english-named buttons. And we don't really need a link for issues, they are still a part of github. Just my opinion ;)

the2pizza commented on index.html:
```diff
@@ -15,46 +15,58 @@
+      <div class="footer-wrapper">
+        <section class="footer-content">
+          <p class="white-text">Vive la résistance<br>
+            Обратная связь - <a class="main__links white-text" href="https://t.me/kira2pizza">Telegram</a>
```
Please shift the block to left side

the2pizza commented:
Please delete service files and use .gitignore for workspaces.xml

the2pizza commented:
@nizhgo perfectly done btw

nizhgo commented:
Changed the page in accordance with the feedback on Twitter

The names of social networks are now in English

Removed the link to github issues

Added .gitignor and deleted workspaces.xml

After changing the page, it looks like this: https://ib.co/bsJC90t

(I added black icons with links, but they didn't look particularly organic on the page. Didn't add them)

theyllneverfindyourgrave commented:
@the2pizza wants the text back and move footer to the left side

nizhgo commented:
ok

how about placing contacts in the footer:
https://ibb.co/bXRzQ0Y
or just as there is in the right?


Should I also return the title?
